examples/b2c/checkout/retrigger-place-order/README.md

```diff
@@ -0,0 +1,22 @@
+# Re Trigger Place Order Flow
+
+The implementation provided in this directory contains flow metadata files that you can deploy to any Salesforce B2B or B2B2C Commerce enabled org. Use these files to set up an end-to-end Re-Trigger Place Order flow that will let the Commerce Admin Re-trigger the Place Order Flow once he fixes the issues with Place Order process
+
+
+##  The advantages of using a ReTrigger Place Order Flow:
+
+1. It will let the Commerce Admin Re-trigger the Reference Apex API when he fixes any errors with the Cart to Order Summary Async Process.
+
+
+## Deploy
+Note: Create through Workbench, no UI setup available
```

Comment:
Should we add notes to both the b2b and b2c README that the flow can also be installed via SFDX (please put in the SFDX command as well). Also mention that if installed using sfdx the flow will need to be activated after install.

Reply:
Updated the README